### PR TITLE
release: 3.6.6

### PR DESCRIPTION
**Pull request by AhyoungRyu**
### Feat:
* Add `customExtensionParams` for `sdk.addSendbirdExtensions` (#698)
  The 3rd parameter customData to the `sdk.addSendbirdExtension` function, allowing it to be delivered from outside of UIKit React.
  e.g.
  ```
  <SendbirdProvider
    customExtensionParams={{
      a: 'a', // the key-value set will be passed when sdk.addSendbirdExtensions is called
    }}
  />
  ```
* Call `sdk.addSendbirdExtensions` during the connection process (#682)

### Fixes:
* Change the MessageInput cursor style from disabled to not-allowed; Thanks @roderickhsiao (#697)
* PendingMsg is missing isUserMessage method (#695)
  This resolves the issue where spreading the message object in the reducer loses some methods like `isUserMessage` and `isFileMessage`
* fix util functions logic of verifying message type. We updated logic in util functions to verify the message type. (#700)

### Chore:
* Update Trunk-Based Development to Scaled Trunk-Based Development (#696)
  It describes the flow with short-lived feature branches, code review, and build automation before integrating into main.

### CODE REVIEW

**Comment by sendbird-sdk-deployment** [comment]
[Creating Ticket] 🔖 Done https://sendbird.atlassian.net/browse/SDKRLSD-935

**Review by sravan-s — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,28 @@
 # Changelog - v3
 
+## [v3.6.6] (Aug 3 2023)
+### Feat:
+* Add `customExtensionParams` for `sdk.addSendbirdExtensions` (#698)
+  The 3rd parameter customData to the `sdk.addSendbirdExtension` function, allowing it to be delivered from outside of UIKit React.
+  e.g.
+  ```
+  <SendbirdProvider
+    customExtensionParams={{
```
Comment (sravan-s) [inline CHANGELOG.md:10]: minor suggestion, its better to memoize this object in this release note,
```
const memoizedCustomExtensionParams = useRef({ ... })
<SendbirdProvider
    customExtensionParams={memoizedCustomExtensionParams}
``` 
There is chance customers might shoot themselves in the foot later

Comment (AhyoungRyu) [reply CHANGELOG.md]: https://github.com/sendbird/sendbird-uikit-react/pull/699/commits/e245e7c3a92d8cd4eaeab5bd4c9aedc9d028e3b6 Yeah thanks for catching it out!

**Comment by AhyoungRyu** [comment]
Rebased after #700 merged

**Pull request merged**